File: app/src/main/java/com/ptt/libdemo/java_only/MainActivity.java
package com.ptt.libdemo.java_only;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import com.ptt.libdemo.R;
import com.ptt.mediaquery.MediaQuery;

@SuppressLint("Registered")
public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        MediaQuery.get().getImageRepository().queryAlbum();
    }
}

File: app/src/main/java/com/ptt/libdemo/kotlin_only/MainActivity.kt
package com.ptt.libdemo.kotlin_only

import android.annotation.SuppressLint
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.ptt.libdemo.R
import com.ptt.mediaquery.MediaQuery

@SuppressLint("Registered")
class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        MediaQuery.get().imageRepository.queryAlbum()
    }
}
